packages/migrate/bin/summarise-report-values.js

```diff
@@ -2,7 +2,7 @@ const fs = require('fs');
 
 // the properties that can be in a new report
 const properties = Object.keys(
-  require('../lib/models').Report.schema.tree.diff
+  require('../src/models').Report.schema.tree.diff
```

Comment:
This line causes an error for me - the file seems to have been moved to the `@neontribe/gbptm-loodb` package.

Reply:
Correct; I'll point it in the right place else find a better way to source the property keys.